documentation/modules/adding-source-provider.adoc

```diff
@@ -21,35 +21,150 @@ You can add {a-rhv} source provider by using the {ocp} web console.
-* *vCenter password*: vCenter user password
-* *VDDK init image*: VDDKInitImage path
+* *Provider resource name*: Name to display in the list of providers.
+* *URL*: Endpoint.
+* *VDDK init image*: VDDKInitImage path.If left empty, some functionality will not be available.
+* *Username*: vCenter user, for example, `user@vsphere.local`.
+* *Password*: vCenter user password.
+* *SHA-1 fingerprint*: [TBD].
+
+. Choose one of the following options for validating a CA certificate:
 
-. To allow a migration without validating the provider's CA certificate, select the *Skip certificate validation* check box. By default, the checkbox is cleared, meaning that the certificate will be validated.
-. Enter the *SHA-1 fingerprint*.
+** *Skip validation* : Migrate without validating a CA certificate.
+** *Use the system CA certiicate*: Migrate after validating the system CA certificate.
+** *Use a custom CA certificate*: Migrate after validating a custom CA certificate.
+
+.. To skip validation, select the *Skip certificate validation* check box.
+.. To validate the system CA certfiticate, leave the *Skip certificate validation* check box cleared and leave the *CA certificate* text box empty.
+.. To validate a custom CA certificate, leave the *Skip certificate validation* check box cleared and _either_ drag the CA certificate to the text box _or_ browse for it and click *Select*.
 endif::[]
 ifdef::rhv[]
-. Select *Red Hat Virtualization* from the *Provider type* list.
+. Click *Red Hat Virtualization*
+. Specify the following fields:
+thats
+* *Provider resource name*: Name to display in the list of providers
+* *URL*: Endpoint
+* *Username*: Username
+* *Password*: Password
+
+. Choose one of the following options for validating a CA certificate:
+
+** *Skip validation* : Migrate without validating a CA certificate.
+** *Use the system CA certiicate*: Migrate after validating the system CA certificate.
+** *Use a custom CA certificate*: Migrate after validating a custom CA certificate.
+
+.. To skip validation, select the *Skip certificate validation* check box.
+.. To validate the system CA certfiticate, leave the *Skip certificate validation* check box cleared and leave the *CA certificate* text box empty.
+.. To validate a custom CA certificate, leave the *Skip certificate validation* check box cleared and _either_ drag the CA certificate to the text box _or_ browse for it and click *Select*.
+endif::[]
+ifdef::ova[]
+. Click *Open Virtual Appliance (OVA)*.
 . Specify the following fields:
 
-* *Provider name*: Name to display in the list of providers
-* *RHV Manager host name or IP address*: {manager} host name or IP address -  if a certificate for FQDN is specified, the value of this field needs to match the FQDN in the certificate
-* *RHV Manager user name*: {manager} user
-* *RHV Manager password*: {manager} password
+* *Provider resource name*: Name to display in the list of providers
+* *URL*: OVA server endpoint
```

Comment:
that a bit misleading since we're the ones that set up the OVA server, it should rather be the URL to the NFS-share that serves the OVA(s)

Reply:
@yaacov -- for UI, also.

Reply:
@ahadas -- How's this? "URL of the NFS file share that serves the OVA"

Reply:
@ahadas Is the suggested wording OK?